File: Python/stage2/S2_FOR_03.py
# S2_FOR_03 — Sum until threshold
def sum_until(nums, threshold):
   total = 0
   for n in nums:
       if total + n > threshold:
           break
       total += n
   return total
print(sum_until([1, 2, 3, 4, 5], 7))

File: js/stage2/S2_FOR_03.js
// S2_FOR_03 — Sum until threshold
function sumUntil(nums, threshold) {
  let sum = 0;
  for (const n of nums) {
    if (sum + n > threshold) break;
    sum += n;
  }
  return sum;
}
// Tests
console.log(sumUntil([1, 2, 3, 4, 5], 7));
console.log(sumUntil([5, 5, 5], 6));
